.gitignore

```diff
@@ -9,8 +9,9 @@ bin
 # Test binary, build with `go test -c`
```

Comment:
Top level comment: Consider adding a section to [local-development section of our makefile documentation](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/makefile.md#local-development). Perhaps something like:

```md
### `make test/coverage`

Outputs a filtered version of the each package's unit test coverage profiling via go's coverage tool to a local `coverage.html` file.
```

Reply:
Thanks - added ✅